### PR TITLE
Added akka http dependency to amqp transport

### PR DESCRIPTION
**Pull request by pielas**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -107,6 +107,7 @@ lazy val amqpTransport = (project in file("rhttpc-amqp")).
     name := "rhttpc-amqp",
     libraryDependencies ++= {
       Seq(
+        "com.typesafe.akka"        %% "akka-http"                     % akkaHttpV,
```
Comment (arkadius) [inline build.sbt]: Why this change? rhttpc-amqp is client protocol agnostic. It only handle passing requests thru amqp.

Comment (pielas) [reply build.sbt]: I didn't notice that this library is needed only to perform tests, I added this dependency only to test
